onnxruntime/test/providers/cpu/tensor/scatter_op_test.cc

```diff
@@ -174,30 +167,14 @@ static void scatter_indices_updates_dont_match(const char* op_name, int op_versi
   test.AddInput<int64_t>("indices", {1, 3}, {1, 3, 3});
   test.AddInput<float>("updates", {1, 2}, {1.1f, 2.1f});
   test.AddOutput<float>("y", {1, 5}, {1.0f, 1.1f, 3.0f, 2.1f, 5.0f});
-  test.Run(OpTester::ExpectResult::kExpectFailure, "Indices vs updates dimensions differs at position=1 3 vs 2", {kTensorrtExecutionProvider});
+  test.Run(OpTester::ExpectResult::kExpectFailure, "", {kTensorrtExecutionProvider});
```

Comment:
curious why we remove the error message here?

Reply:
When I changed the CUDA kernel, I thought we don't need the check al the dims one by one, but just assert indices_shape == update_shape. But I didn't change the CPU kernel so the error message is different between CUDA and CPU. Now I added them back to CUDA.